crates/bevy_ecs/src/component.rs

```diff
@@ -93,6 +93,42 @@ use std::{
 /// [`Table`]: crate::storage::Table
 /// [`SparseSet`]: crate::storage::SparseSet
 ///
+/// # Adding component's hooks
+///
+/// See [`ComponentHooks`] for a detailed explanation of component's hooks.
```

Comment:
Oh actually this documentation might be better on `ComponentHooks` (didn't realize it existed as a separate thing). What do you think?

Reply:
Hard to say actually. Maybe I could mention the `Component` macro's way of defining hooks?